Add settings to connect to Salesforce MyDomain url (#28)
logstash-plugins/logstash-input-salesforce: docs/index.asciidoc

SOQL statement. Additional fields can be filtered on by
adding field1 = value1 AND field2 = value2 AND...

[id="plugins-{type}s-{plugin}-sfdc_instance_url"]
===== `sfdc_instance_url`

  * Value type is <<string,string>>
  * There is no default value for this setting.

> The url of a salesforce instance. Provide this if you already
> during login want to connect to your instance url instead of
> `login.salesforce.com` or `test.salesforce.com`.

Reviewer: ```suggestion
The url of a Salesforce instance. Provide the url if you want to connect to your Salesforce instance instead of
`login.salesforce.com` or `test.salesforce.com` at login.
```

Reviewer: I added some minor wording tweaks for your consideration. Please see what you think and make sure that I didn't alter your intended message with the suggested rewording.

Author: Thank you! I feel like each time I get a chance to work with somebody that is more experienced at writing tech documentation is a huge learning opportunity for me. Very much appreciated!

> 

Reviewer 2: Can we please add a note to the effect that either this setting or `use_test_sandbox` can be used, not both

Reviewer: I agree that this would be helpful to users.  It would be nice to add this info to the `use_test_sandbox` option, too.

Author: Thank you for the suggestion - yes, that makes a lot of sense.
I have added a sentence (also below for `use_test_sandbox`).
What do you think?

Reviewer: Nicely done. Thank you!  ❤️
